stagecoach/jobs/publishers/publish-all/app.py

```diff
@@ -64,6 +64,9 @@ async def _handler(assets):
     )
     _entries = await publisher_client.fetch()
     response = await publisher_client.publish_many(_entries)
+    print("hashes:")
```

Comment:
Nit: f“Published data with tx hashes: {‘, ‘,join([hex(res.hash) for res in response])}”